taker/src/main.rs

```diff
@@ -315,6 +315,12 @@ async fn main() -> Result<()> {
                 tracing::info!(endpoint = %http_endpoint, "HTTP interface is ready");
             })
         }))
+        .attach(AdHoc::on_request("Rocket request", |req, _data| {
+            Box::pin(async move {
+                let request = req;
+                tracing::debug!(%request,  "rocket");
```

Comment:
My comment would have applied to both fairings :sweat_smile:

Reply:
> My comment would have applied to both fairings 😅

indeed! we should come up with a scheme that avoids this unfortunate duplication :( doing a follow-up now!

Reply:
https://github.com/itchysats/itchysats/pull/1565 - I should have done it properly in the first place ;-)